node/pkg/driver/device_connectivity/device_connectivity_iscsi_test.go

```diff
@@ -199,13 +199,15 @@ func TestGetMpathDevice(t *testing.T) {
 			fake_executer := mocks.NewMockExecuterInterface(mockCtrl)
 			fake_helper := mocks.NewMockOsDeviceConnectivityHelperIscsiInterface(mockCtrl)
 			lunId := 0
-			arrayIdentifier := "X"
-			path := strings.Join([]string{"/dev/disk/by-path/ip*", "iscsi", arrayIdentifier, "lun", strconv.Itoa(lunId)}, "-")
-			fake_helper.EXPECT().WaitForPathToExist(path, 5, 1).Return(
-				tc.waitForPathToExistReturn.devicePaths,
-				tc.waitForPathToExistReturn.exists,
-				tc.waitForPathToExistReturn.err,
-			)
+			arrayIdentifiers := []string{"X"}
```

Comment:
I think that what we need is one test for one iSCSI name and a separate test to having 2 or more iSCSI names.

Reply:
yes, I will add test cases about one more iscsi names